src/api/layer/quickbooks.ts

```diff
@@ -0,0 +1,47 @@
+import { StatusOfSyncFromQuickbooks } from "../../types/quickbooks";
+import { get, post } from "./authenticated_http";
+
+export const syncFromQuickbooks = post<
+  Record<string, unknown>,
+  Record<string, unknown>,
+  { businessId: string }
+>(({ businessId }) => `/v1/businesses/${businessId}/quickbooks/sync-from`);
+
+export const statusOfSyncFromQuickbooks = get<
+  { data: StatusOfSyncFromQuickbooks },
+  { businessId: string }
+>(
+  ({ businessId }) => `/v1/businesses/${businessId}/quickbooks/sync-from/status`
+);
+
+export const initQuickbooksOAuth = post<
+  {
+    data: {
+      type: "Quickbooks_Authorization_Params";
```

Comment:
I don't think you ned to include this in the body of the request

Reply:
The naming is a bit confusing, but this is actually the response body